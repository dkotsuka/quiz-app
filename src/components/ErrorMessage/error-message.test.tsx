import React from 'react'
import { ErrorPlaceHolder } from '.'
import {create} from 'react-test-renderer';

describe("ErrorPlaceHolder", () => {
    it("matches snapshot", () => {
        const divider = create(<ErrorPlaceHolder />)
        expect(divider).toMatchSnapshot()
    })
})