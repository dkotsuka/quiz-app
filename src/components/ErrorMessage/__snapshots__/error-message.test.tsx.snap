// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPlaceHolder matches snapshot 1`] = `
Array [
  <View
    height={160}
    style={
      Array [
        Object {
          "height": 160,
        },
      ]
    }
  />,
  <Text
    style={
      Array [
        Object {
          "fontFamily": "Roboto Bold",
          "fontSize": 20,
        },
        Object {
          "color": "gray",
        },
      ]
    }
  >
    An error has occurred. Please, try again later.
  </Text>,
]
`;
